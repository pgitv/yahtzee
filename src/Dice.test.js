import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Dice from './Dice';

it('renders shallowly without crashing', function() {
  shallow(<Dice handleClick={null} locked={[false]} dice={[1]} />);
});

it('passes snapshot test UNlock', function() {
  let wrapper = shallow(
    <Dice handleClick={null} locked={[false]} dice={[1]} />
  );
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});

it('passes snapshot test Locked', function() {
  let wrapper = shallow(<Dice handleClick={null} locked={[true]} dice={[1]} />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
