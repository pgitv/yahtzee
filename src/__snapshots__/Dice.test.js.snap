// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passes snapshot test Locked 1`] = `
<div
  className="Dice"
>
  <Die
    handleClick={null}
    idx={0}
    key="0"
    locked={true}
    val={1}
  />
</div>
`;

exports[`passes snapshot test UNlock 1`] = `
<div
  className="Dice"
>
  <Die
    handleClick={null}
    idx={0}
    key="0"
    locked={false}
    val={1}
  />
</div>
`;
